IdentityCore/src/broker_operation/response/MSIDBrokerOperationAccountResponse.m

```diff
@@ -0,0 +1,65 @@
+//
+//  MSIDBrokerOperationAccountResponse.m
+//  IdentityCore iOS
+//
+//  Created by JZ on 10/1/19.
+//  Copyright © 2019 Microsoft. All rights reserved.
+//
+
+#import "MSIDBrokerOperationAccountResponse.h"
+#import "MSIDAccount+MSIDJsonSerializable.h"
+
+@implementation MSIDBrokerOperationAccountResponse
+
+#pragma mark - MSIDJsonSerializable
+
+- (instancetype)initWithJSONDictionary:(NSDictionary *)json error:(NSError **)error
```

Comment:
would get accounts also return device info?

Reply:
Yes, it should. The device info will be part of the base class because all response will include that.
The device info will be done in a separate PR :)